tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
+                 "dockerskeleton" \
+                 "nodejs6action"  \
+                 "python2action"  \
+                 "python3action"  \
+                 "swift3action"   \
```

Comment:
I tried looking for base swift image we could use, I would not want to build a base image in our repo, as it would not save us anytime. I would preferred to take a swift linux image already ready, and we just add the whisk proxy code.

But I think this is orthogonal and should not block this PR, it can be done in another PR once we get @paulcastro help find such base image, maybe from kitura or cloudfoundry pack